python/lsst/ts/ess/csc/rpi_data_client.py

```diff
@@ -417,7 +431,7 @@ async def run_command(self, command: str, **parameters: Any) -> None:
         """
         json_str = json.dumps({"command": command, "parameters": parameters})
         await asyncio.wait_for(
-            self._basic_run_command(json_str), timeout=COMMUNICATE_TIMEOUT
+            self._basic_run_command(json_str), timeout=RPiDataClient.communicate_timeout
```

Comment:
This can be `self.communicate_timeout` and I think that would be clearer (the reader doesn't have to ask "isn't think already the RPiDataClient class?) and more idiomatic.

Not all languages allow accessing class constants as if they were instance variables, but Python does.

Reply:
Thanks, I didn't know that. Updated.